Updates to readme file.

andreamussap opened the pull request:
@wickedest - updates to the readme.

waldyrious commented on README.md:
```diff
@@ -2,19 +2,23 @@
 
 ### Installation via webpack
-The recommended way to install mergely is to use npm and [webpack](https://webpack.js.org/) to install mergely and its dependencies.  It is highly recommended that you start by cloning [mergely-webpack](https://github.com/wickedest/mergely-webpack).  It has everything that you need to get started.
+The recommended way to install mergely and its dependencies is to use a Node package manager (npm) and [webpack](https://webpack.js.org/).
```
Just a small suggestion:

```suggestion
The recommended way to install mergely and its dependencies is to use a Node package manager (`npm` or `yarn`) and [webpack](https://webpack.js.org/).
```

waldyrious commented on README.md:
```diff
@@ -45,7 +51,7 @@ $(document).ready(function () {
 ### Synchronous content initialization
 
-The following example can be used to set the `lhs` and `rhs` editors synchronously (i.e. their contents are already known):
+The following example can be used to set the `lhs` and `rhs` editors synchronously, that is, when their contents are already known):
```
You removed the opening parenthesis, so the closing one is unbalanced. It should be restored:

```suggestion
The following example can be used to set the `lhs` and `rhs` editors synchronously (that is, when their contents are already known):
```

waldyrious commented on README.md:
```diff
@@ -2,19 +2,23 @@
 ### Installation via .tgz
 
-Unpack mergely.tgz into a folder, e.g. `./lib`, and then add the following to the `<head>` of your target html source file.
+Unpack mergely.tgz into a folder, for example, `./lib`, and add the following to the `<head>` of your target HTML source file.
```
Minor nit: There should be no comma after "for example".

waldyrious commented on README.md:
```diff
@@ -243,7 +249,7 @@ $('#mergely').mergely('options');
 ### resize
 
-Resize the editor.  Must be called explicitly if `autoresize` is disabled.
+Resize the editor. It must be called explicitly if `autoresize` is disabled.
```
```suggestion
Resizes the editor. It must be called explicitly if `autoresize` is disabled.
```

Note that the `### search` section below also needs to be adjusted to match verbal tense ("Searches...")

waldyrious commented on README.md:
```diff
@@ -337,7 +343,7 @@ $('#mergely').mergely('unmarkup');
 ### update
 
-Manual update; recalculates diff and applies new settings.
+Manual update, recalculates the diff and applies new settings.
```
The `;` to `,` is not appropriate — it introduces a run-on sentence.

Maybe this could be reworked into, e.g.

> Manually updates the editor by recalculating the diff and applying new settings

WDYT?

wickedest replied:
```suggestion
Manually updates the editor by recalculating the diff and applying new settings.
```

wickedest commented on README.md:
```diff
@@ -2,19 +2,23 @@
 http://mergely.com
 
-Mergely is a javascript component to diff/merge files interactively in a browser, providing rich API that enables you to easily integrate Mergely into your existing web application.  It is suitable for comparing text files online, e.g. .txt, .html, .xml, .c, .cpp, .java, etc.  Mergely has a javascript implementation of the Longest Common Subsequence diff algorithm (LCS) and a customizable markup  ...
+Mergely is a javascript component to differencing and merging files interactively in a browser, providing rich API that enables you to easily integrate Mergely into your existing web application. It is suitable for comparing text files online, for example, .txt, .html, .xml, .c, .cpp, .java, etc.  
```
There is something funky with the verb "to differencing and merging", and it also loses the technical jargon "diff", which is important to retain.

```suggestion
Mergely is a JavaScript component for differencing and merging files interactively in a browser (diff/merge), providing rich API that enables you to easily integrate Mergely into your existing web application. It is suitable for comparing text files online, for example, .txt, .html, .xml, .c, .cpp, .java, etc.  
```

andreamussap replied:
Thanks for the tech explanation :)

wickedest commented on README.md:
```diff
@@ -2,19 +2,23 @@
-Mergely is a javascript component to diff/merge files interactively in a browser, providing rich API that enables you to easily integrate Mergely into your existing web application.  It is suitable for comparing text files online, e.g. .txt, .html, .xml, .c, .cpp, .java, etc.  Mergely has a javascript implementation of the Longest Common Subsequence diff algorithm (LCS) and a customizable markup  ...
+Mergely is a javascript component to differencing and merging files interactively in a browser, providing rich API that enables you to easily integrate Mergely into your existing web application. It is suitable for comparing text files online, for example, .txt, .html, .xml, .c, .cpp, .java, etc.  
+
+Mergely has a javascript implementation of the Longest Common Subsequence (LCS) diff algorithm, and a customizable markup engine.
```
```suggestion
Mergely has a JavaScript implementation of the Longest Common Subsequence (LCS) diff algorithm, and a customizable markup engine.
```

wickedest commented on README.md:
```diff
@@ -2,19 +2,23 @@
 ### Installation via .tgz
 
-Unpack mergely.tgz into a folder, e.g. `./lib`, and then add the following to the `<head>` of your target html source file.
+Unpack mergely.tgz into a folder, for example, `./lib`, and add the following to the `<head>` of your target HTML source file.
```
```suggestion
Unpack mergely.tgz into a folder, for example `./lib`, and add the following to the `<head>` of your target HTML source file.
```

andreamussap replied:
https://www.thefreedictionary.com/for+example

waldyrious replied:
None of those examples apply to this case, IIUC. But that's a minor nit, really, so no need to bikeshed it :)

andreamussap commented:
> @andreamussap thanks - I had a few suggestions to apply, also thanks to @waldyrious for his suggestions. Also, please add this to the CHANGES.md:
> 
> ```lisp
> ## 4.0.13
> * patch: updated documentation
> * patch: fixed typos
> ```

This is done, PR sent :)
